src/System.Runtime/tests/DBNullTests.InvalidCast.cs

```diff
@@ -0,0 +1,50 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.Tests
+{
+    public static partial class DBNullTests
```

Comment:
These tests fail the nutc compile step on ilc, so it's not at test execution time. I do believe that it is temporarily, so I'll talk to PN folks to try to find out why is that failing, but I wanted to unblock running the rest of the tests for now.